web/modules/custom/itk_activity/src/Form/Multistep/MultistepFormBase.php

```diff
@@ -118,16 +119,68 @@ protected function getData() {
   }
 
   /**
+   * Get the progress bar array.
+   *
+   * @return array
+   */
+  protected function getProgressBar() {
+    return [
+      'items' => [
+        [
+          'title' => \Drupal::translation()->translate("About activity"),
+          'href' => Url::fromRoute('itk_activity.multistep_about')->toString(),
+          'open' => TRUE,
+        ],
+        [
+          'title' => \Drupal::translation()->translate("Information"),
+          'href' => Url::fromRoute('itk_activity.multistep_information')->toString(),
+          'open' => $this->store->get('step1'),
+        ],
+        [
+          'title' => \Drupal::translation()->translate("Categories"),
+          'href' => Url::fromRoute('itk_activity.multistep_categories')->toString(),
+          'open' => $this->store->get('step2'),
+        ],
+        [
+          'title' => \Drupal::translation()->translate("Image"),
+          'href' => Url::fromRoute('itk_activity.multistep_image')->toString(),
+          'open' => $this->store->get('step3'),
+        ],
+        [
+          'title' => \Drupal::translation()->translate("Details"),
+          'href' => Url::fromRoute('itk_activity.multistep_details')->toString(),
+          'open' => $this->store->get('step4'),
+        ],
+        [
+          'title' => \Drupal::translation()->translate("Confirm"),
+          'href' => Url::fromRoute('itk_activity.multistep_confirm')->toString(),
+          'open' => $this->store->get('step5'),
+        ],
+      ],
+    ];
+  }
+
+  /**
+   * Accept a step in the form.
```

Comment:
So what is this: https://github.com/aakb/genlydaarhus/blob/develop/web/themes/custom/genlyd_aarhus/templates/misc/status-messages.html.twig